src/components/Wallet/WalletDeposit.jsx

```diff
@@ -76,24 +75,12 @@ class WalletDeposit extends React.Component {
 
     clearInterval(depositTimer);
     depositTimer = setInterval(async () => {
-      let utxos;
+      let updates;
       try {
-        utxos = await fetchAddressUTXOs(this.state.address, network, client);
```

Comment:
Another place where functionality was folded into the implementation of `fetchAddressUTXOs`